### PR TITLE
Conform function headers to coding guidelines

### PR DESCRIPTION
**Pull request by Phyma**
**When merged this pull request will:**
- Fix every header on all files that i have on my fork
- Tested with Silentspikes python script #5253
- Even if the other thing is not accepted this is still good
- Alot of things that i have done is switch to current header, add example, add public and fixed "ReturnValue" to "Return Value"

I believe this is a label "impact/trivial" because it's only headers

### CODE REVIEW

File: addons/fcs/functions/fnc_onForceUpdate.sqf [hunk 1]
```diff
@@ -1,3 +1,18 @@
+/*
+ * Author: ACE
+ *
+ *
+ * Arguments:
+ * Nothing
```
Comment (jonpas) [inline addons/fcs/functions/fnc_onForceUpdate.sqf]: This should be `None`, I noticed it was wrong in some other files before as well.

Comment (SilentSpike) [inline addons/fcs/functions/fnc_onForceUpdate.sqf]: The script will now complain about this too after latest commit 😄

Comment (Phyma) [inline addons/fcs/functions/fnc_onForceUpdate.sqf]: Yeay. Well we can put this as WIP so I can fix this in the same tempo that silentspike updates the script and we just do both at the same time :)

I was just bored and didnt feel like scripting so did this.

**Comment by Phyma** [comment]
Done! No more errors! Plus new version of the software only public functions

**Comment by bux** [comment]
DO IT NAOW!

**Pull request merged**